great_expectations/util.py

```diff
@@ -406,29 +400,6 @@ def gen_directory_tree_str(startpath: PathStr):
     return output_str
 
 
-# NOTE: Can delete once CLI is removed
-def lint_code(code: str) -> str:
-    """Lint strings of code passed in.  Optional dependency "black" must be installed."""
```

Comment:
@cdkini @billdirks 

I'm not sure why we ever needed this.
Maybe a long time ago we weren't enforcing the `black` format style in our CI, so we had to rely on this?